app/reducers/ProductsReducer.jsx

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios'
+
+const initialState = {
+  products: [{name: 'blah', imageURL: 'whatever', price: 10000}]
+}
+
+// CONSTANTS
+export const LOAD_PRODUCTS = 'LOAD_PRODUCTS'
+
+const getProducts = (res) => {
+  return {
+    type: LOAD_PRODUCTS,
+    payload: res.data
+  }
+}
+
+// ACTIONS
```

Comment:
Don't forget to res.data your axios requests so you don't get a ton of extra metadata!